site/ja/tfx/guide/examplegen.md

```diff
@@ -0,0 +1,139 @@
+# ExampleGen TFX パイプラインコンポーネント
+
+Note: これらのドキュメントは私たちTensorFlowコミュニティが翻訳したものです。コミュニティによる
+翻訳は**ベストエフォート**であるため、この翻訳が正確であることや[英語の公式ドキュメント](https://www.tensorflow.org/?hl=en)の
+最新の状態を反映したものであることを保証することはできません。
+この翻訳の品質を向上させるためのご意見をお持ちの方は、GitHubリポジトリ[tensorflow/docs](https://github.com/tensorflow/docs)にプルリクエストをお送りください。
+コミュニティによる翻訳やレビューに参加していただける方は、
+[docs-ja@tensorflow.org メーリングリスト](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ja)にご連絡ください。
+
+TFX パイプラインコンポーネントの1つである ExampleGen コンポーネントはデータを TFX のパイプラインに投入します。
+これは外部のファイルやサービスからデータを読み取り、ほかの TFX のコンポーネントが読み取る Examples を生成します。
+また、、一貫性のある方法でデータセットを分割します。分割は設定により変更可能です。
+同時に機械学習のベストプラクティスに従い、データセットの並び替えを行います。
+
+- 入力: CSV や `TFRecord`, BigQuery といった外部のデータソースからのデータ
+- 出力: `tf.Example` レコード
+
+## ExampleGen とその他のコンポーネント
+
+ExampleGen は [TensorFlow Data Validation](tfdv.md) ライブラリを利用するコンポーネントにデータを提供します。
+これには [SchemaGen](schemagen.md), [StatisticsGen](statsgen.md), [Example Validator](exampleval.md) が含まれます。
+また、[TensorFlow Transform](tft.md) を利用する [Transform](transform.md) にもデータを提供し、最終的には推論時にデプロイメントターゲットへとデータを供給します。
+
+## ExampleGen コンポーネントの使い方
+
+サポートされるデータソース (現在、CSV ファイル、TF Example フォーマットの TFRecord ファイル、BigQuery のクエリ結果、の3つがサポートされています) を用いる場合、ExampleGen コンポーネントは典型的には非常に簡単にデプロイ可能で、ほとんど改修を必要とせずに利用できます。
+典型的なコードは次のようになります:
+
+```python
+from tfx.utils.dsl_utils import csv_input
+from tfx.components.example_gen.csv_example_gen.component import CsvExampleGen
+
+examples = csv_input(os.path.join(base_dir, 'data/simple'))
+example_gen = CsvExampleGen(input_base=examples)
+```
+
+次のように、外部の TF Example 形式のファイルを直接読み込むこともできます:
+
+```python
+from tfx.utils.dsl_utils import tfrecord_input
+from tfx.components.example_gen.import_example_gen.component import ImportExampleGen
+
+examples = tfrecord_input(path_to_tfrecord_dir)
+example_gen = ImportExampleGen(input_base=examples)
+```
+
```

Comment:
[question] If I take a look at the original markdown, there is a section you didn't translate in the PR.

Maybe the original document has been changed after you translated it?

![image](https://user-images.githubusercontent.com/1390726/69046616-c48b4b00-0a3c-11ea-960e-649486ece1fe.png)

https://github.com/tensorflow/tfx/blob/master/docs/guide/examplegen.md

Reply:
Yeah, this PR is outdated at the moment of your review. It's now updated to the latest 👍